### PR TITLE
PortfolioMobile

### PR DESCRIPTION
**Pull request by sheylaPozo**
_

In this Project I worked from the last project "Porfolio" and added some sections, images, backgrounds, and modifications, it's been done with HTML5, and CSS.

Also the purpose was to understand how to parse a Figma design to create a UI, use Flexbox to place elements in the page, use CSS Grid to place elements in the page, and build a personal portfolio sit

_

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,65 @@
...
+![Demo](https://user-images.githubusercontent.com/54015740/120832021-cc7ae500-c525-11eb-9812-ca2a95e061bc.jpg)
+
+---
+The main purpose of the project [Portfolio](https://sheylapozo.github.io/PortfolioMobile/) was for me to develop some hands on work as a code reviewer, using my previous knokledge working with Html, CSS, and linters, which also serves as practice in documenting my work in a professional way, with the learning objectives of using the best practices in HTML code, semantic HTML tags, CSS selectors in a correct way, CSS box model, and to perform a code review for myself on a past project for experience.
+-------
+---
+
+## 💖 Live Demo
+
+[Portfolio](https://sheylapozo.github.io/PortfolioMobile/)
```
Comment (jubaan) [inline README.md:32]: - [ ] The README is the most important part of a repository, we should always make sure that the links are working properly and if we have a link to a live preview of our project we should ensure is working as expected. 
  Right now the Live Preview is only showing the title PortfolioMobile title
- [ ] Let's ensure the project follows the design and is a close version of the original.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta name="author" content="Sheyla Pozo" />
+  <title>PortfolioMobile</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" />
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,300;0,700;1,200&display=swap"
+    rel="stylesheet" />
+  <link rel="stylesheet" href="assets/style.css" />
```
Comment (jubaan) [inline index.html:13]: - [ ] Let's use this sintax when using URLs that link files within our repository `./directory/of/file`(don't miss the dot at the start).

File: index.html [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+</head>
+<body>
+  <main>
+    <header>
+      <a href="#">My logo</a>
+      <nav>
+        <i class="fas fa-bars"></i>
+      </nav>
+    </header>
```
Comment (jubaan) [inline index.html:22]: - [ ] An HTML has 3 main containers `<header>`, `<main>`, and `<footer>`. these are use individually and they should not contain each other. Let's remove the `header` from the `main` container, and put it in a better place just before the `main` section.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+    <section class="section-top">
+      <h1 class="section-title">I’m Sheyla Glad to see you!</h1>
```
Comment (jubaan) [inline index.html:24]: - [ ] There should be only one `<h1>` title tag per page. This improves SEO, and the following titles should follow the next level of `<hx>`h1, h2, h3... h6` without skipping levels. Let's modify our titles to use only one h1 per page, and update the other titles with the corresponding title level.

**Review by jubaan — CHANGES_REQUESTED** [review]
Hi @sheylaPozo , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (sheylaPozo) [reply index.html:24]: All the required and optional changes are adapted into the project now, I added the improvements recommended by the code reviewer, thank you so much.

File: index.html (hunk 3 above)
Comment (sheylaPozo) [reply index.html]: thanks

File: index.html (hunk 2 above)
Comment (sheylaPozo) [reply index.html]: alright improved, thank you

File: README.md (hunk 1 above)
Comment (sheylaPozo) [reply README.md]: thank you

File: index.html [hunk 5]
```diff
@@ -0,0 +1,184 @@
...
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="butonns-languages">
+          <li class="btn-lg">html</li>
+          <li class="btn-lg">css</li>
+          <li class="btn-lg">javaScript</li>
+        </ul>
+        <button type="button" class="btn-project">See Project</button>
+      </li>
```
Comment (ZawadiSumba66) [inline index.html:69]: - [ ] Please lets make sure that we follow the design as much as possible. The card for each project has a white background from the template given.
 yours:
image
template:
image

**Review by ZawadiSumba66 — CHANGES_REQUESTED** [review]
Hi @sheylaPozo, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**